src/client.ts

```diff
@@ -74,40 +74,56 @@ export default class UnleashClient extends EventEmitter {
     feature: FeatureInterface | undefined,
     context: Context,
     fallback: Function,
-  ): boolean {
+  ): [boolean, VariantDefinition | undefined] {
```

Comment:
in addition to boolean true/false was want to know the actual variant that the strategy resolved to if it was provided.